src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/StringValidators.java

```diff
@@ -45,10 +45,6 @@ public static boolean isValidCharacters(String toTest, String qualifiedFieldName
 
 
     /**
-     * The email regex is taken from OWASP https://owasp.org/www-community/OWASP_Validation_Regex_Repository
-     * ^[a-zA-Z0-9_+&*-]+(?:\.[a-zA-Z0-9_+&*-]+)*@(?:[a-zA-Z0-9-]+\.)+[a-zA-Z]{2,15}$
-     * Sonar has highlighted issues with the * and + grouping quantifiers causing backstepping (recursion)
-     * so we are modifying them to be ++ and *+ (possessive = no backstepping)
      * @param email
```

Comment:
you can remove the comments completely for isValidEmailAddress, as having the params etc on their own don't really add anything.

Reply:
actually, similar to Martin's comments in the Web changes, add a comment saying this regex is same as the ch-account service one

Reply:
Updated with the tests used in the web